src/main/java/seedu/address/ui/FocusCard.java

```diff
@@ -205,4 +219,18 @@ private void setProfilePicture(Name candidateName) {
         stackPane.setClip(circle);
         stackPane.setAlignment(Pos.CENTER);
     }
+
+    private void setSchedule() {
+        if (interview != null) {
+            DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd LLLL yyyy");
+            String temp = interview.getInterviewDate().getDayOfWeek().toString();
+            String d = temp.charAt(0) + temp.substring(1).toLowerCase() + ",";
+            day.setText(d);
+            date.setText(interview.getInterviewDate().format(formatter));
+            time.setText("@ " + interview.getInterviewStartTime().toString());
+            scheduleMessage.setText(SCHEDULE_MESSAGE);
```

Comment:
En En nice job! Only comment is I actually thought that it's 1 `Label` for all 3 fields. But no issue at all! Darn good job!